### PR TITLE
feature/add reshape2 tools

### PR DESCRIPTION
**Pull request by djcomlab**
Add reshape2 tools, `melt` and `cast`.

### CODE REVIEW

**Comment by djcomlab** [comment]
Need to set up new Jenkins build configs for `phnmnl/container-reshape2-melt` and `phnmnl/container-reshape2-cast` in order to be able to test (UOXF team currently has no access to Jenkins).

**Comment by djcomlab** [comment]
This is ready to go now.

File: config/tool_conf.xml [hunk 1]
```diff
@@ -317,6 +317,8 @@
       <tool file="phenomenal/w4m/transformation_config.xml" labels="W4M"/>
       <tool file="phenomenal/w4m/batch_correction.xml" labels="W4M"/> 
       <tool file="phenomenal/w4m/NmrNormalization_xml.xml" labels="W4M"/>
+      <tool file="phenomenal/reshape2/melt.xml" labels="reshape2"/>
```
Comment (ilveroluca) [inline config/tool_conf.xml]: I think the `labels` attribute doesn't belong here.  The tool label comes from the tool's xml definition.  Looking at the Galaxy docs it looks like only `labels="updated"` is valid and has a special meaning:
    https://docs.galaxyproject.org/en/master/admin/tool_panel.html#sections-labels

Comment (djcomlab) [reply config/tool_conf.xml]: Should these be updated across the board? I just followed the convention that was already used in `tool_conf.xml` for the other tools.

**Pull request merged**